Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -51,8 +51,8 @@
 
 upgradeProperties[2017] = {
     '2017' : {
-        'Geom' : 'DB:Extended',
-        'GT' : 'auto:phase1_2017_realistic',
+        'Geom' : 'Extended2017dev',
+        'GT' : 'auto:phase1_2017_hcaldev',
```

Comment:
The 2017dev geometry itself does not need validation, but it can only be used (for DIGI-RECO) with an appropriate global tag having HCAL conditions for all the new channels. As we just got one, it could not have been used in the workflow until now. @ianna doesn't have the latest version of it in the database yet, so I assigned the xml version for now.

This shouldn't "interfere" with the new pixel geometry PRs, as those are creating a separate workflow for validation. @davidlange6 requested that I not make yet another separate workflow for HCAL 2017.

At some point, all of these geometry changes need to be reconciled into one final 2017 geometry, but I think that should be done after or otherwise outside of this PR.